sdk/core/core-rest-pipeline/CHANGELOG.md

```diff
@@ -1,12 +1,6 @@
 # Release History
 
-## 1.9.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.9.2 (2022-09-01)
```

Comment:
should this be v1.10 since it has a behavioral change albeit minor?

Reply:
I am fine with either way.  @xirzec any thoughts?